From 4b76ddfb3f9328010a90e56fef02ce2d9abcb007 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 16 Aug 2016 10:11:41 +0000
Subject: [PATCH 219/462] MANTIS 0042283 extension: integration of ingenico
 payment device for KIS kiosk

added tempus.jar
contains generated classes from schema file


git-svn-id: svn://localhost/SelfServiceBuild/trunk@331 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/build.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index 969bdad..270a5d6 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -24,7 +24,7 @@
 
     <!-- Definition der Verzeichnisse mit den Java-Modulen ======= -->
     <property name="CommonDirList"   value="${CommonJavaDir}/OpenORB-1.3.1/OpenORB"/>
-    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/OPI_CCV20120201,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/ApplicationAdapter,${PlatformJavaDir}/AdminApplication,${PlatformJavaDir}/SSDMApplication,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/SOAP-Settings,${PlatformJavaDir}/CLAApplet,${PlatformJavaDir}/DSGEventReporting"/>
+    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/Ingenico-Tempus,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/OPI_CCV20120201,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/ApplicationAdapter,${PlatformJavaDir}/AdminApplication,${PlatformJavaDir}/SSDMApplication,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/SOAP-Settings,${PlatformJavaDir}/CLAApplet,${PlatformJavaDir}/DSGEventReporting"/>
     <property name="AblsDirList"     value="${AblsJavaDir}/KioskProxy,${AblsJavaDir}/TunnelServer"/>
     <property name="AppDirList"      value=""/>
 
-- 
2.41.0.windows.1

